geost/base.py

```diff
@@ -1110,6 +1110,65 @@ def to_qgis3d(
         qgis3d = self._create_geodataframe_3d(relative_to_vertical_reference)
         qgis3d.to_file(outfile, driver="GPKG", **kwargs)
 
+    def to_kingdom(self, outfile, vw=1500.0, vs=1600.0):
```

Comment:
typing staat ook in de functiedefinitie:

`def to_kingdom(self, outfile: str | WindowsPath, vw: float = 1500.0, vs: float = 1600.0`